persistence/src/main/scala/akka/persistence/kafka/journal/KafkaJournal.scala

```diff
@@ -102,6 +102,13 @@ class KafkaJournal(config: Config) extends AsyncWriteJournal { actor =>
     Resource.liftF(cassandraClusterOf)
   }
 
+  def jsValueCodec(config: KafkaJournalConfig): JsValueCodec = {
```

Comment:
please make it F[JsValueCodec]

Reply:
I don't get how that would improve anything